### PR TITLE
Ivan skadorva

### PR DESCRIPTION
**Pull request by IvanSkadorva**
Created the footer component

### CODE REVIEW

File: src/app/footer/footer.component.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<footer>
+        <div class="container__footer">
```
Comment (marinbonibon) [inline src/app/footer/footer.component.html:2]: Think about class names, bro =)

File: src/app/footer/footer.component.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+                  <li><a href="#">About Us</a></li>
+                  <li><a href="#">Q&A</a></li>
+                </ul>
+              </div>
+            </div>
+            <div class="container__social">
+              <div class="container__social-apple">
+                <ul>
+                  <li><div><a href="#"><img src="assets/images/App-Store-Outline-badge.svg"></a></div></li>
+                </ul>
```
Comment (marinbonibon) [inline src/app/footer/footer.component.html:28]: It's redundant, you may make it easier

File: src/app/footer/footer.component.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
+<footer>
+        <div class="container__footer">
+            <div class="container__links">
+              <h1>BROWSE</h1>
```
Comment (marinbonibon) [inline src/app/footer/footer.component.html:4]: Is the word 'Browse' the most important word on your page?

File: src/app/footer/footer.component.scss [hunk 4]
```diff
@@ -0,0 +1,61 @@
+
+
+
+footer{
+  font-family: SF Pro Display;
```
Comment (marinbonibon) [inline src/app/footer/footer.component.scss:5]: Start with the font you want, and always end with a generic family, to let the browser pick a similar font in the generic family, if no other fonts are available.

**Review by avmaxim — CHANGES_REQUESTED** [review]
@IvanSkadorva Подпулить дев забыл) Пока не смотрю твой ПР) Как подольешь - маякни)

**Pull request closed without merging**